src/services/print.js

```diff
@@ -214,6 +220,15 @@ ngeo.Print.prototype.encodeMap_ = function(map, scale, object) {
   let layers = this.ngeoLayerHelper_.getFlatLayers(mapLayerGroup);
   layers = layers.slice().reverse();
 
+  // (1.) We loop on every layers and if one of them doesn't use native angle,
+  // this set the global value to use for the print (default is true)
+  layers.forEach((layer) => {
+    if (layer.getProperties()['useNativeAngle'] === false) {
```

Comment:
Shouldn't be 'printNativeAngle' To be clear that it's used only by the print?

Reply:
@sbrunner You're right about that. I renamed it.